.github/workflows/search.yml

```
@@ -0,0 +1,267 @@
+              f.write(html)
+
+          print(f"✅ 搜索完成！共找到 {len(result)} 个本子")
+          print("📄 纯文本 HTML 已生成: search_result.html")
+          EOF
+
+      - name: 上传 HTML 文件
+        uses: actions/upload-artifact@v4
+        with:
+          name: search_${{ github.event.inputs.keyword }}_page${{ github.event.inputs.page }}
+          path: search_result.html          keyword = "${{ github.event.inputs.keyword }}"
+          page = int("${{ github.event.inputs.page }}")
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)